### DISCUSSION TITLE
[Suggestion] Set State/color wiring inputs for neon lights

### DISCUSSION THREAD

**Discussion by Steamlined** [FakeFishGames/Barotrauma#10020] [Ideas and suggestions]
Title. I thought about making some dixie tube displays on my custom sub, but then found you can't wire neon lights. 
I just think it'd be neat.

**Comment by MaaiG** [comment]
I’ve never met a neon lamp with RGB lighting ;). That’s [not possibl](https://en.wikipedia.org/wiki/Neon_lamp)e. Neon lamps can be of different colors, but they cannot change color.

**Reply by Steamlined** [reply]
Honestly I was mainly concerned about the setting state wiring input anyway
